mod/invitefriends/start.php

```diff
@@ -36,20 +35,7 @@ function invitefriends_page_handler($page) {
 		return false;
 	}
 	
-	elgg_set_context('friends');
-	elgg_set_page_owner_guid(elgg_get_logged_in_user_guid());
-
-	$title = elgg_echo('friends:invite');
-
-	$body = elgg_view('invitefriends/form');
-
-	$params = array(
-		'content' => $body,
-		'title' => $title,
-	);
-	$body = elgg_view_layout('one_sidebar', $params);
-
-	echo elgg_view_page($title, $body);
+	echo elgg_view_resource('invitefriends');
```

Comment:
Let's use invitefriends/index or something so it's easier to extend later if needed